pages/common/html5validator.md

```diff
@@ -0,0 +1,32 @@
+# html5validator
+
+> A command line tool for testing HTML5 validity.
+> More information: <https://github.com/svenkreiss/html5validator>.
+
+- Validate a specific file:
+
+`html5validator {{path/to/file}}`
+
+- Validate all HTML files in a specific directory:
+
+`html5validator --root {{path/to/directory}}`
+
+- Show warnings as well as errors:
+
+`html5validator --show-warnings {{path/to/file}}`
+
+- Match file patterns using a glob:
+
+`html5validator --root {{path/to/directory}} --match "{{*.html *.php}}"`
+
+- Ignore specific directory names:
+
+`html5validator --root {{path/to/directory}} --blacklist {{node_modules}}`
```

Comment:
How are multiple directory names specified in this example?

Reply:
Ah, fair point. They are space-separated. So it should probably have quotes around "{{node_modules}}".

Maybe something like:
```suggestion
`html5validator --root {{path/to/directory}} --blacklist "{{node_modules vendor}}"`
```

Reply:
Yeah let's do that.